DUI3-DX/Connectors/Rhino/Speckle.Connectors.Rhino7/Plugin/Speckle.Connectors.Rhino7Plugin.cs

```diff
@@ -28,16 +27,9 @@ public class SpeckleConnectorsRhino7Plugin : PlugIn
   protected override string LocalPlugInName => "Speckle (New UI)";
   public AutofacContainer? Container { get; private set; }
 
-  public SpeckleConnectorsRhino7Plugin()
-  {
-    Instance = this;
-  }
-
-#nullable disable
   ///<summary>Gets the only instance of the Speckle_Connectors_Rhino7Plugin plug-in.</summary>
-  public static SpeckleConnectorsRhino7Plugin Instance { get; private set; }
+  public static SpeckleConnectorsRhino7Plugin Instance { get; } = new();
```

Comment:
I think this is actually not going to work

`Instance` should be the instance of the plugin that the Rhino host app creates, 
We should not be newing a plugin object ourselves.
I think the code was fine before.